lib/matplotlib/backends/backend_ps.py

```diff
@@ -27,36 +21,24 @@ def _fn_name(): return sys._getframe(1).f_code.co_name
 from matplotlib.cbook import is_string_like, get_realpath_and_stat, \
     is_writable_file_like, maxdict, file_requires_unicode
 from matplotlib.figure import Figure
-
 from matplotlib.font_manager import findfont, is_opentype_cff_font
 from matplotlib.ft2font import FT2Font, KERNING_DEFAULT, LOAD_NO_HINTING
 from matplotlib.ttconv import convert_ttf_to_ps
 from matplotlib.mathtext import MathTextParser
 from matplotlib._mathtext_data import uni2type1
-from matplotlib.text import Text
 from matplotlib.path import Path
 from matplotlib import _path
 from matplotlib.transforms import Affine2D
-
 from matplotlib.backends.backend_mixed import MixedModeRenderer
 
-
-import numpy as np
-import binascii
-import re
 try:
```

Comment:
This try except can go too. All versions of python that we support have set in the global namespace